Update README.md (#56)
UBC-MDS/PyXplor: README.md

`pyxplor` is a comprehensive Python package designed to automate and streamline the Exploratory Data Analysis (EDA) process. Tailored for various data types including numeric, categorical, binary, and time series data, `pyxplor` aims to enhance data interpretation through a suite of specialized plotting functions. This package seeks to reduce the complexity and time invested in initial data analysis, making it an essential tool for data scientists and analysts at all levels.

## Installation (developers)

Note that the package is still under development.

> - create an environment with conda (then activate the environment conda activate <env-name>)

Reviewer: ```suggestion
1. Create an environment with conda  and then activate the environment.
```bash
conda create -n pyxplor
conda activate pyxplor
```
```

```bash
conda create -n pyxplor
conda activate pyxplor

> 

Reviewer: ```suggestion
2. Install poetry inside the environment.
3. Run `poetry install`.
```

`pyxplor` is a comprehensive Python package designed to automate and streamline the Exploratory Data Analysis (EDA) process. Tailored for various data types including numeric, categorical, binary, and time series data, `pyxplor` aims to enhance data interpretation through a suite of specialized plotting functions. This package seeks to reduce the complexity and time invested in initial data analysis, making it an essential tool for data scientists and analysts at all levels.

## Installation (developers)

Note that the package is still under development.

> For testing environment: 

Reviewer: ```suggestion
```

`pyxplor` is a comprehensive Python package designed to automate and streamline the Exploratory Data Analysis (EDA) process. Tailored for various data types including numeric, categorical, binary, and time series data, `pyxplor` aims to enhance data interpretation through a suite of specialized plotting functions. This package seeks to reduce the complexity and time invested in initial data analysis, making it an essential tool for data scientists and analysts at all levels.

## Installation (developers)

Note that the package is still under development.

> 1. Create an environment with conda  and then activate the environment.
> ```bash
> conda create -n pyxplor
> conda activate pyxplor
> - Install poetry inside the environment (conda install poetry)
> - Run poetry install
> - Then you can test it with pytest tests

Reviewer: ```suggestion
1. Create an environment with conda and then activate the environment.
```bash
conda create -n pyxplor
conda activate pyxplor
```
2. Install poetry inside the environment.
```bash
conda install poetry
```
3. Run poetry install
```bash
poetry install
```
```

> ## Installation

Reviewer: ```suggestion
## Installation (developers)
```

> Note that the package isn't complete yet.

Reviewer: ```suggestion
Note that the package is still under development.
```
